.github/workflows/molecule_tests.yml

```diff
@@ -9,6 +9,7 @@ jobs:
         role:
           - deploy_user
           - passenger
+          - python
```

Comment:
this is already in main, fwiw

Reply:
I pulled it in from main; not sure why it's getting flagged as new?